### DISCUSSION TITLE
Testing Plugin with Flexget Test Fixtures

### DISCUSSION THREAD

**Discussion by kingamajick** [Flexget/Flexget#3618] [Q&A]
Flexget has a series of [test fixtures ](https://github.com/Flexget/Flexget/blob/develop/flexget/tests/conftest.py), is it possible to use these to test a plugin outside of Flexget as it doesn't seem to be configured as a plugin per [PyTests instructions](https://docs.pytest.org/en/7.1.x/how-to/writing_plugins.html)

**Comment by gazpachoking** [comment]
Hmm. Not sure it's appropriate that we provide our fixtures as a formal plugin. Is it possible to import our conftest from yours, or specify it manually as a plugin in your conftest? Might your plugin be useful enough to others to submit it as a PR to our main repo?

**Reply by kingamajick** [reply]
The plugin Is pretty unique so wouldn't have any value in submitting as a PR, I'll see if I can pull it in via conftest

**Reply by gazpachoking** [reply]
Will this work in your conftest? `pytest_plugins = "flexget.tests.conftest"`

**Reply by kingamajick** [reply]
Thanks I'll give that a try. I've actually be re-examining my workflow, and I think it may actually be possible to do it with standard flexget plugins (however I'm running into some issues with the SFTP plugins, but I'll raise a bug against that)
